types/ink-divider/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for ink-divider 2.0
+// Project: https://github.com/juresotosek/ink-divider#readme
+// Definitions by: omjadas <https://github.com/me>
```

Comment:
<!--
  1 failure:  This line should ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ink-divider/index.d.ts;
  Line: 3;
-->
- :no_entry_sign: This line should have your github username in it, not /me